src/Cities/Domain/Models/City.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Cities\Domain\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * @property int                             $id
+ * @property string                          $name
+ * @property \Illuminate\Support\Carbon|null $created_at
+ * @property \Illuminate\Support\Carbon|null $updated_at
+ *
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|City newModelQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|City newQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|City query()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|City whereCreatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|City whereId($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|City whereName($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|City whereUpdatedAt($value)
+ *
+ * @mixin \Eloquent
+ */
+class City extends Model
+{
+    protected $fillable = [
+        'name',
+    ];
```

Comment:
instead of defining fillable, use $guarded so you dont have to change the fillable array each time a new attribute is added to the model.